### PR TITLE
Add dark mode toggle in documentation.

### PR DESCRIPTION
**Pull request by swastkk**
Reference- https://github.com/nexB/vulnerablecode/pull/1177

Signed-off-by - swastik <swastkk@gmail.com>

### CODE REVIEW

**Comment by swastkk** [comment]
@TG1999 can merge this PR instead .

File: docs/source/conf.py [hunk 1]
```diff
@@ -63,3 +63,7 @@
 html_static_path = []
 
 master_doc = "index"
+
+# user starts in light mode
+
+default_dark_mode = False
```
Comment (AyanSinhaMahapatra) [inline docs/source/conf.py:69]: Yes, this is what we want, light stays as default, as that is the case without this plugin.

**Review by AyanSinhaMahapatra — APPROVED** [review]
LGTM! Thanks! I wonder though whether we should add the same to https://github.com/nexB/skeleton, we should have this everywhere.

File: docs/source/conf.py (hunk 1 above)
Comment (swastkk) [reply docs/source/conf.py:69]: > Yes, this is what we want, light stays as default, as that is the case without this plugin.

Yep ✨

**Comment by swastkk** [comment]
Will add this in the skeleton as soon as possible ✨🚀

**Comment by TG1999** [comment]
@swastkk I am closing this PR in favour of https://github.com/nexB/skeleton/commit/5ab9b3a15e0095a8186ca3870cb9f9eade83833d, thanks again for raising this PR

**Pull request closed without merging**